semver/increment.ts

```diff
@@ -161,7 +161,7 @@ export function increment(
       // 1.2.1 bumps to 1.3.0
       if (
         version.patch !== 0 ||
-        version.prerelease.length === 0
+        version.prerelease?.length === 0
```

Comment:
Ditto. This seems introducing a bug. Please add test case to exercise this pattern with prerelease = `undefined`.

Reply:
fixed